app/code/community/Hps/Transit/etc/system.xml

```diff
@@ -203,6 +203,44 @@
                             <comment><![CDATA[The amount of time (in minutes) before recent failures are ignored.]]></comment>
                             <depends><enable_anti_fraud>1</enable_anti_fraud></depends>
                         </fraud_velocity_timeout>
+                        <avs_cvv_validation translate="label">
+                            <label>AVS/CVV Validation</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>67</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                            <depends><enable_anti_fraud>1</enable_anti_fraud></depends>
+                            <comment><![CDATA[Address Verification Service (AVS) and Cardholder Verification Value (CVV) are 
+                            	simple service offered by the cardholders’ issuing banks. With Heartland’s payment gateway, 
+                            	you can elect to send automatic reversals for undersired AVS / CVV result codes, freeing you 
+                            	to focus on your core business logic. As an added bonus, the inclusion of AVS and CVV 
+                            	information during the request helps your transactions qualify for lower Interchange rates, 
+                            	further helping to reduce your overall cost of payment acceptance..]]></comment>
+                        </avs_cvv_validation>
+                        <avs_decline_codes translate="label">
+                            <label>AVS Decline Condition</label>
+                            <frontend_type>multiselect</frontend_type>
+                            <source_model>hps_transit/source_avsresultcodes</source_model>
```

Comment:
Should be: 
`<source_model>hps_transit/source_avsResultCodes</source_model>`

Reply:
updated the model name